### PR TITLE
Release note: New @faststore/core package version

### PR DESCRIPTION
**Pull request by Mariana-Caetano**
#### Types of changes
- [x] New content (guides, endpoints, app documentation)
- [ ] Improvement (make a documentation even better)
- [ ] Fix (fix a documentation error)
- [ ] Spelling and grammar accuracy (self-explanatory)

Atualização do `navigation.json`: https://github.com/vtexdocs/devportal/pull/739

### CODE REVIEW

File: docs/release-notes/2024-05-28-faststore-core-new-version.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+2. Access the VTEX Admin and go to **Headless CMS > FastStore**. You should be able to see the three new content types listed there.
+3. Click one of the content types, and then`Add section` ( `+`).
+4. Choose the `EmptyState` section and update its fields according to your store's requirements.
+
+</details>
+
+<details>
+<summary>Enhanced API extensions</summary>
+
+Follow the [Best practices for using API extensions](https://developers.vtex.com/docs/faststore/api-extensions-best-practices) guide for more information.
```
Comment (Mariana-Caetano) [inline docs/release-notes/2024-05-28-faststore-core-new-version.md:78]: Link quebrado, pois a documentação aestá [neste PR](https://github.com/vtexdocs/dev-portal-content/pull/1175).

File: docs/release-notes/2024-05-24-faststore-core-new-version.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
+title: "FastStore: New @faststore/core package version"
+slug: "2024-05-24-faststore-core-new-version"
+type: "new"
+excerpt: "Upgrade your `@faststore/core` package version to leverage enhanced API extensions, customizable error pages, and support for Next.js 13."
+createdAt: "2024-05-23T10:30:00.000Z"
+---
+
+FastStore v2 users can now update their`@faststore/core` package version and benefit from enhanced API extensions, streamlined error page customization in the Headless CMS, and support for Next.js 13.
```
Comment (carolinamenezes) [inline docs/release-notes/2024-05-24-faststore-core-new-version.md:9]:
```suggestion
FastStore v2 users can now update the `@faststore/core` package version and benefit from enhanced API extensions, streamlined error page customization in the Headless CMS, and support for Next.js 13.
```

File: docs/release-notes/2024-05-24-faststore-core-new-version.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+> For more information on this update, check out [FastStore GitHub releases](https://github.com/vtex/faststore/releases/tag/v3.0.0).
+
+## What has changed?
+
+### Enhanced API extensions
+
+To reduce bundle sizes, align with GraphQL community standards, and get up to date with the latest bundling and compilation tools, the latest `@faststore/core` package version 3.0.0 deprecates the `@faststore/graphql-utils` package in favor of the [`client-preset`](https://the-guild.dev/graphql/codegen/plugins/presets/preset-client) plugin.
```
Comment (carolinamenezes) [inline docs/release-notes/2024-05-24-faststore-core-new-version.md:17]:
```suggestion
To reduce bundle sizes, align with GraphQL community standards, and get up to date with the latest bundling and compilation tools,`@faststore/core` 3.x deprecates the `@faststore/graphql-utils` package in favor of the [`client-preset`](https://the-guild.dev/graphql/codegen/plugins/presets/preset-client) plugin.
```

File: docs/release-notes/2024-05-24-faststore-core-new-version.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+### Streamlined error page customization in the Headless CMS
+
+Before, FastStore had a more limited scope for customizing content on pages such as Login, 404 Error, and 500 Error within the Headless CMS. Customizing these pages was only possible through code, which limited users’ ability to tailor these pages to their specific needs within the Headless CMS.
+
+Now, these new content types are part of the latest `@faststore/core` package version 3.0, and users can customize them within the Headless CMS.
+
+[gif](/)
```
Comment (carolinamenezes) [inline docs/release-notes/2024-05-24-faststore-core-new-version.md:25]: Atualizar antes de mergear

File: docs/release-notes/2024-05-24-faststore-core-new-version.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+### Enhanced API extensions
+
+To reduce bundle sizes, align with GraphQL community standards, and get up to date with the latest bundling and compilation tools, the latest `@faststore/core` package version 3.0.0 deprecates the `@faststore/graphql-utils` package in favor of the [`client-preset`](https://the-guild.dev/graphql/codegen/plugins/presets/preset-client) plugin.
+
+### Streamlined error page customization in the Headless CMS
+
+Before, FastStore had a more limited scope for customizing content on pages such as Login, 404 Error, and 500 Error within the Headless CMS. Customizing these pages was only possible through code, which limited users’ ability to tailor these pages to their specific needs within the Headless CMS.
+
+Now, these new content types are part of the latest `@faststore/core` package version 3.0, and users can customize them within the Headless CMS.
```
Comment (carolinamenezes) [inline docs/release-notes/2024-05-24-faststore-core-new-version.md:23]:
```suggestion
These content types are now part of the `@faststore/core` 3.x, and users can customize them within the Headless CMS.
```

File: docs/release-notes/2024-05-24-faststore-core-new-version.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+In your local FastStore project, install the latest version of `@faststore/core`:
+
+ ```bash
+yarn add @faststore/core@ˆ3.0.0
+
+```
+
+Run the project locally with `yarn dev`.
+
+After running `yarn dev`, update each topic mentioned before as follows:
```
Comment (carolinamenezes) [inline docs/release-notes/2024-05-24-faststore-core-new-version.md:45]: Cortou?

File: docs/release-notes/2024-05-24-faststore-core-new-version.md
Comment (carolinamenezes) [inline docs/release-notes/2024-05-24-faststore-core-new-version.md:1]: Excluir esse arquivo?

File: docs/release-notes/2024-06-25-faststore-core-new-version.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+
+FastStore v2 users can now update their`@faststore/core` package version and benefit from enhanced API extensions, streamlined error page customization in the Headless CMS, and support for Next.js 13.
+
+> ℹ️ For more information on this update, check out [FastStore GitHub releases](https://github.com/vtex/faststore/releases/tag/v3.0.0).
+
+## What has changed?
+
+### Enhanced API extensions
+
+To reduce bundle sizes, align with GraphQL community standards, and get up to date with the latest bundling and compilation tools, the latest `@faststore/core` package version 3.0.0 deprecates the `@faststore/graphql-utils` package in favor of the [`client-preset`](https://the-guild.dev/graphql/codegen/plugins/presets/preset-client) plugin.
```
Comment (carolinamenezes) [inline docs/release-notes/2024-06-25-faststore-core-new-version.md:18]:
```suggestion
To reduce bundle sizes, align with GraphQL community standards, and get up to date with the latest bundling and compilation tools, `@faststore/core` 3.x deprecates the `@faststore/graphql-utils` package in favor of the [`client-preset`](https://the-guild.dev/graphql/codegen/plugins/presets/preset-client) plugin.
```

File: docs/release-notes/2024-06-25-faststore-core-new-version.md [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+
+### Streamlined error page customization in the Headless CMS
+
+Before, FastStore had a more limited scope for customizing content on pages such as Login, 404 Error, and 500 Error within the Headless CMS. Customizing these pages was only possible through code, which limited users’ ability to tailor these pages to their specific needs within the Headless CMS.
+
+Now, these new content types are part of the latest `@faststore/core` package version 3.0, and users can customize them within the Headless CMS.
```
Comment (carolinamenezes) [inline docs/release-notes/2024-06-25-faststore-core-new-version.md:24]:
```suggestion
These content types are now part of `@faststore/core` 3.x, and users can customize them within the Headless CMS.
```

File: docs/release-notes/2024-06-25-faststore-core-new-version.md [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+<summary>Migrating to Next.js 13</summary>
+
+1. Open your store’s `package.json` file and, in `dependencies`, edit the `next` entry:
+
+   ```bash
+   "dependencies": {
+   ...
+   "next": "^13.5.6",
+   ...
+   },
```
Comment (carolinamenezes) [inline docs/release-notes/2024-06-25-faststore-core-new-version.md:58]:
```suggestion
   "dependencies": {
      ...
      "next": "^13.5.6",
      ...
   },
```

File: docs/release-notes/2024-06-25-faststore-core-new-version.md [hunk 10]
```diff
@@ -0,0 +1,80 @@
...
+
+</details>
+
+<details>
+<summary>Error page customization in the Headless CMS</summary>
+
+1. To sync the updated version, open a new terminal and run `faststore cms-sync`.
+2. Access the VTEX Admin and go to **Headless CMS > FastStore**. You should be able to see the three new content types listed there.
+3. Click one of the content types, and then`Add section` ( `+`).
+4. Choose the `EmptyState` section and update its fields according to your store's requirements.
```
Comment (carolinamenezes) [inline docs/release-notes/2024-06-25-faststore-core-new-version.md:71]: Adicionaria apenas o link para consultar a doc sobre esse tema

**Review by carolinamenezes — CHANGES_REQUESTED** [review]

File: docs/release-notes/2024-06-28-faststore-core-new-version.md [hunk 11]
```diff
@@ -0,0 +1,80 @@
...
+
+</details>
+
+<details>
+<summary>Error page customization in the Headless CMS</summary>
+
+1. To sync the updated version, open a new terminal and run `faststore cms-sync`.
+2. Access the VTEX Admin and go to **Headless CMS > FastStore**. You should be able to see the three new content types listed there.
+3. Click one of the content types, and then`Add section` ( `+`).
+4. Choose the `EmptyState` section and update its fields according to your store's requirements.
```
Comment (Mariana-Caetano) [inline docs/release-notes/2024-06-28-faststore-core-new-version.md:71]:
```suggestion
4. Choose the [`EmptyState`](https://developers.vtex.com/docs/guides/faststore/organisms-empty-state) section and update its fields according to your store's requirements.
```

File: docs/release-notes/2024-06-28-faststore-core-new-version.md [hunk 12]
```diff
@@ -0,0 +1,80 @@
+---
+title: "FastStore: New @faststore/core package version"
+slug: "2024-06-28-faststore-core-new-version"
```
Comment (Mariana-Caetano) [inline docs/release-notes/2024-06-28-faststore-core-new-version.md:3]:
```suggestion
slug: "2024-08-19-faststore-core-new-version"
```

**Review by carolinamenezes — APPROVED** [review]

File: docs/release-notes/2024-06-28-faststore-core-new-version.md [hunk 13]
```diff
@@ -0,0 +1,80 @@
+---
+title: "FastStore: New @faststore/core package version"
+slug: "2024-08-19-faststore-core-new-version"
+type: "new"
+excerpt: "Upgrade your `@faststore/core` package version to leverage enhanced API extensions, customizable error pages, and support for Next.js 13."
+createdAt: "2024-05-23T10:30:00.000Z"
+updateAt: "2024-06-26T10:30:00.000Z"
```
Comment (Mariana-Caetano) [inline docs/release-notes/2024-06-28-faststore-core-new-version.md:7]:
```suggestion
updateAt: "2024-08-19T10:30:00.000Z"
```

**Pull request merged**